src/components/Select/Select.stories.tsx

```diff
@@ -1,5 +1,7 @@
 import type { Meta, StoryObj } from '@storybook/react';
-import { Select } from '~/src/index';
+import { useState } from 'react';
+import { Select, SelectSingle } from '~/src/index';
+import type { SelectOption, SelectProperties } from './Select';
 import { SingleSelectOptions } from './testUtils';
 
 const meta: Meta<typeof Select> = {
```

Comment:
A couple of the displayed props in the Story have no effect on the component since it has been changed to render the `SelectSingle`
- isMulti
- maxSelections

```suggestion
const meta: Meta<typeof SelectSingle> = {
```